### PR TITLE
 DEV-87 Docs: add information on sleep limits

### PR DESCRIPTION
**Pull request by sylwiavargas**
There's a recurring question about sleep maximum time on Discord.

This PR is adding information on the sleep limits on both the steps guide, and two reference pages (step.sleep, step.sleepUnti).

### CODE REVIEW

File: pages/docs/guides/delayed-functions.mdx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { CodeGroup } from "src/shared/Docs/mdx";
 
 # Enqueueing future jobs
 
-You can easily enqueue jobs in the future with Inngest. Inngest offers two ways to run jobs in the future: delaying jobs for a specific amount of time (eg. run in 1 day), or running code at a specific date and time. There are some benefits to enqueuing jobs using Inngest:
+You can easily enqueue jobs in the future with Inngest. Inngest offers two ways to run jobs in the future: delaying jobs for a specific amount of time (up to a year, and for free plan up to seven days), or running code at a specific date and time. There are some benefits to enqueuing jobs using Inngest:
```
Comment (djfarrelly) [inline pages/docs/guides/delayed-functions.mdx:5]: We can add this copy on the limits page then link to pricing from the limits page. https://www.inngest.com/docs/usage-limits/inngest

I'm thinking the limits page says "Sleep, delays, etc. up to X for paid plans and Y for free plans" Check the pricing page for more information.

**Review by djfarrelly — APPROVED** [review]
Suggestion to add and link to the limits doc

File: pages/docs/usage-limits/inngest.mdx [hunk 2]
```diff
@@ -11,20 +11,23 @@ Some of these limits are customizable, so if you need more than what the current
 
 The following applies to `step` usage.
 
-**Timeout**
+### Sleep duration
 
-Each step has a timeout depending on the hosting provider of your choice ([see more info][provider-docs]),
-but Inngest supports up to `2 hours` at the maximum.
+Sleep (with `step.sleep()` and `step.sleepUntil()`) up to a year, and for free plan up to seven days. Check the [pricing page](/pricing) for more information.
```
Comment (sylwiavargas) [inline pages/docs/usage-limits/inngest.mdx:16]: @djfarrelly is this what you had in mind? 👀

File: pages/docs/guides/delayed-functions.mdx (hunk 1 above)
Comment (sylwiavargas) [reply pages/docs/guides/delayed-functions.mdx:5]: Great suggestion, see https://github.com/inngest/website/pull/821/files#r1661356650

**Pull request merged**